src/txn/txn_nsnap.c

```diff
@@ -43,13 +43,13 @@ __nsnap_drop_one(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *name)
 	/* Bump the global ID if we are removing the first entry */
 	if (found == TAILQ_FIRST(&txn_global->nsnaph)) {
 		WT_ASSERT(session, !__wt_txn_visible_all(
-		    session, txn_global->nsnap_oldest_id));
+		    session, txn_global->nsnap_oldest_id, NULL));
```

Comment:
Note that named snapshots and timestamps *can* coexist, but I can't think of a reason why an application would want both.